google-cloud-spanner/src/test/java/com/google/cloud/spanner/SessionPoolTest.java

```diff
@@ -1638,7 +1637,6 @@ public Void call() {
         .containsEntry(MetricRegistryConstants.NUM_ACQUIRED_SESSIONS, 3L);
     assertThat(record.getMetrics())
         .containsEntry(MetricRegistryConstants.NUM_RELEASED_SESSIONS, 3L);
-    assertThat(record.getMetrics()).containsEntry(MetricRegistryConstants.IN_USE_SESSIONS, 0L);
```

Comment:
No test for new Gauges?

Reply:
Added test to validate newly added Metric. I had to modify `MetricRegistryTestUtils` to support and test multiple timeseries entries. PTAL when you get a chance.